### DISCUSSION TITLE
Iconos svg con el menor código posible

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#14] [Announcements]
# Iconos svg con el menor código posible

Los iconos SVG son una forma popular y versátil de agregar gráficos vectoriales a un sitio web.

https://www.zkreations.com/2022/12/iconos-svg-con-el-menor-codigo-posible.html

**Comment by gianles** [comment]
gran aporte daniel, sin duda es algo que nos ayudara muchisimo a la hora de optimizar mejor nuestros codigos html y css para ahorrar lineas! un saludo! gracias por el aporte.

**Reply by danieIabel** [reply]
Gracias por pasar y comentar amigo. Espero que sea de utilidad para muchas personas, es un proyecto que planeo mantener y mejorar mucho, y que todos pueden usar sin ninguna limitación, completamente open source.

**Comment by gianles** [comment]
a la final creo que voy a usar estos icons para mi proyectos de blogs! muchas gracias otra vez por meteors @daniel

**Reply by danieIabel** [reply]
Gracias por usarlo amigo, aprovechando el hecho si notas que falta algún icono que consideres necesario no dudes en sugerirlo.

**Reply by gianles** [reply]
excelente, si si, esoty viendo algunos iconos y estan buenos y super cortos su etiqueta svg, lo que si podrias agregar es agregar algunos iconos pero tambien en su estado solido! por ejemplo "heart" que solo es el borde el que se ve, podrias hacerlo slido y que se rellene todo. es solo una idea! muchas gracias por compartirlo!!

**Reply by danieIabel** [reply]
Para esos casos puedes crear una utilidad amigo, agrega este codigo css:

```css
.i-solid { fill: currentColor }
```

Luego al icono le agregas esa clase:

```html
<svg class="i i-heart i-solid" viewBox="0 0 24 24">
  <path d="m12 21-8.8-8.3A5.6 5.6 0 1 1 12 6a5.6 5.6 0 1 1 8.9 6.6z"></path>
</svg>
```

**Reply by gianles** [reply]
Fascinante Daniel. Muchísimas gracias, me a servido. Incluso puedo darle más grosor a las lineas.  De momento tengo todos los iconos al alcance de mis manos

**Comment by danieIabel** [comment]
He actualizado el proyecto a la version 2.0.0. Agregando y mejorando los iconos, incluyendo la página de demostración. También se removió por completo la variante **Pugjs**

**Comment by danieIabel** [comment]
10 Nuevos iconos de la categoría "Music & Video" agregados.
4 De los iconos existentes se mejoraron para lograr mayor consistencia y geometría.
